### ISSUE TITLE
QBChat disconnect completionBlock is not called if chat was already disconnected

### ISSUE THREAD

**Issue by intonarumori** [QuickBlox/quickblox-ios-sdk#632]
**Environment details**
Environment: iOS10 simulator, QuickBlox 2.8.0.1 install with CocoaPods

**Did this work before?**
No idea.

**Expected behavior**
Calling `QBChat.instance().disconnect(...)` multiple times should result in the `completionBlock` being called: with no error on successful disconnect, and possibly with error if it was already disconnected.

**Actual behavior**
Calling `QBChat.instance().disconnect(...)` the second time, the `completionBlock` is not called at all.

**Steps to reproduce the behavior**

- Connect to chat with a valid username and password and wait for a successful connection
- Set breakpoint in disconnect block
- Disconnect from chat, breakpoint is hit
- Call disconnect again, breakpoint is not hit

```
QBChat.instance().disconnect { [weak self] (error) in

            print("DISCONNECT RESULT RECEIVED") // breakpoint here
            if let error = error {
                print("DISCONNECT ERROR \(error)")
            } else {
                print("DISCONNECTED SUCCESSFULLY")
            }
        }
```

**Comment by Raikerian** [comment]
Hi,

Thank you for reporting the issue, it is confirmed to be true. We are expecting a new release later this week or earlier next one.

**Comment by Raikerian** [comment]
Fixed in 2.9.
